common/build/eslint-plugin-fluid/src/test/example/no-unchecked-record-access/fileWithOnlyRecordAccess.ts

```diff
@@ -104,16 +113,35 @@ aImplicitType!.length; // ok: This should be caught by tsc, not by this custom l
 let aLetExpectingString: string = indexedRecordOfStrings.a; // defect: Assigning index property 'a' to a strict string variable, but 'a' might not be present
 let aLetExpectingStringOrUndefined: string | undefined = indexedRecordOfStrings.a; // ok: Assigning index property 'a' to string or undefined variable, 'a' might not be present
 let aLetExpectingStringAfterVariableDeclaration: string;
-aLetExpectingStringAfterVariableDeclaration = indexedRecordOfStrings.a; // defect
-
-interface NonNullableProps {
-	definitelyString: string;
-	maybeString?: string;
-}
+aLetExpectingStringAfterVariableDeclaration = indexedRecordOfStrings.a; // defect: Assigning index property 'a' to a strict string variable, but 'a' might not be present
+let aLetExpectingStringOrUndefinedAfterVariableDeclaration: string | undefined;
+aLetExpectingStringOrUndefinedAfterVariableDeclaration = indexedRecordOfStrings.a; // ok: Assigning index property 'a' to string or undefined variable, 'a' might not be present
 
+/* Non-Nullable Properties */
 const nonNullObj: NonNullableProps = { definitelyString: "hello" };
 nonNullObj.definitelyString.length; // ok: Accessing length of non-nullable property
 nonNullObj.maybeString.length; // ok: This should be caught by tsc, not by this custom lint rule
 
 let possiblyUndefined: string | undefined;
 possiblyUndefined = nonNullObj.maybeString; // ok: Assigning optional property to variable of type 'string | undefined'
+
+/* Static Types */
```

Comment:
Rearrange is good, but I think you lost your comment:
```
 * no-unchecked-record-access should not apply to static types, since they are guaranteed to have the properties they define.
```